SimPPS/PPSSimTrackProducer/python/SimTrackProducerForFullSim_cff.py

```diff
@@ -0,0 +1,34 @@
+import FWCore.ParameterSet.Config as cms
+def customise(process):
+
+  if hasattr(process.VtxSmeared,"X0"):
+        VertexX = process.VtxSmeared.X0
+        VertexY = process.VtxSmeared.Y0
+        VertexZ = process.VtxSmeared.Z0
+
+  if hasattr(process.VtxSmeared,"MeanX"):
+      VertexX = process.VtxSmeared.MeanX
+      VertexY = process.VtxSmeared.MeanY
+      VertexZ = process.VtxSmeared.MeanZ
+
+  process.load("SimG4Core.Application.g4SimHits_cfi")
+  process.g4SimHits.Generator.HepMCProductLabel   = 'LHCTransport'
+  process.g4SimHits.Generator.MinEtaCut        = -13.0
+  process.g4SimHits.Generator.MaxEtaCut        = 13.0
+
+  process.g4SimHits.SteppingAction.MaxTrackTime = cms.double(2000.0)
```

Comment:
@mundim, this may be redefined only temporary. For production it is not good to change global time limits. We need to define a special G4Region for PPS geometry and apply specific time limit for this G4Region only.